addons/pos_loyalty/static/tests/tours/PosLoyaltyTourMethods.js

```diff
@@ -156,6 +156,15 @@ export function pointsAwardedAre(points_str) {
         },
     ];
 }
+export function pointsTotalIs(points_str) {
+    return [
+        {
+            content: 'loyalty points awarded ' + points_str,
+            trigger: '.loyalty-points-total .value:contains("' + points_str + '")',
+            run: function () {}, // it's a check
```

Comment:
We can omit this line ;) if no `run` function is given, it will still do the check.